src/components/UI/Header.js

```diff
@@ -21,20 +21,15 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
-import { Link } from 'react-router-dom';
 import AuthButton from '../Auth/AuthButton';
-import logo from './bcgovlogo.svg';
+import { APP_INFO } from '../../constants';
+import Title from './Title';
```

Comment:
I like the idea of abstracting this into a component but i think the name __Title__ is generic and could be misunderstood. Perhaps something like `Brand` or `Banner`?